packages/augur-state/src/db/AbstractDB.ts

```diff
@@ -66,4 +70,8 @@ export abstract class AbstractDB {
       return false;
     }
   }
+
+  protected destroy(): Promise<void> {
```

Comment:
I'm on the fence about whether or not this is a good idea. I don't know of any use case except testing. I can easily remove this method.

Reply:
Can this be moved into the wrapped class?

Reply:
Defintely.